VirtualFleetDocker.py

```diff
@@ -0,0 +1,372 @@
+import os
+import sys
+import json
+import time
+import socket
+import shutil
+import logging
+import pathlib
+import argparse
+import multiprocessing
+from dataclasses import dataclass, field
+from concurrent.futures import ThreadPoolExecutor
+
+import docker
+from docker.models.containers import Container
+
+
+@dataclass
+class Configuration:
+    number_of_log_last_lines_to_show: int = 5
+    config_dir: str = "./config"
+    tmp_config_dir: str = field(init=False)
+    external_server_config: str = field(init=False)
+    module_gateway_config: str = field(init=False)
+    vernemq_config: str = field(init=False)
+    logs_dir: str = "logs"
+
+    def __post_init__(self):
+        self.tmp_config_dir = os.path.join(self.config_dir, "tmp-configs")
+        self.external_server_config = os.path.join(self.config_dir, "external-server/config.json")
+        self.module_gateway_config = os.path.join(self.config_dir, "module-gateway/config.json")
+        self.vernemq_config = os.path.join(self.config_dir, "vernemq")
```

Comment:
There was probably a miss-communication here, I meant to represent the [virtual-fleet-config.json](https://github.com/bringauto/virtual-fleet/blob/BAF-904/update-virtual-fleet-for-protocol-v2/config/virtual-fleet-config.json) as a dataclass. Basically create a model of the structure so it is better error handled and typed. 

If it is too much work, merge this MR and that can be done later.